cat/CaT-service.yaml

```diff
@@ -2951,12 +2951,9 @@ components:
       type: object
       properties:
         pocurementID:
-          type: string
-          format: uuid
+          $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Schema.yaml#/components/schemas/ProcurementProjectID'
         eventID:
-          type: string
-          description: OCID for the event
-          example: ocds-abc123-00001
+          $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Schema.yaml#/components/schemas/ProcurementEventID'
```

Comment:
@ronin1972 - there is a syntax error in this referenced type (duplicate `example` key) - also, should it now be `ProcurementEventOCID` (not sure what the difference is?)